src/interfaces/coral_web/src/components/Agents/AgentForm.tsx

```diff
@@ -97,3 +97,25 @@ export const AgentForm: React.FC<Props> = ({
     </div>
   );
 };
+
+const RequiredInputLabel: React.FC<{
+  label: string;
+  children: React.ReactNode;
+  className?: string;
+}> = ({ label, children, className }) => (
+  <InputLabel
+    label={
+      <div className="flex items-center gap-x-2 uppercase">
```

Comment:
`styleAs="label"` already has `uppercase`
```suggestion
      <div className="flex items-center gap-x-2">
```